### DISCUSSION TITLE
The file manifest for Kong OSS on K8S with the Kong Manager OSS does not exist

### DISCUSSION THREAD

**Discussion by bilel-zouaghi** [Kong/kong-manager#73] [Q&A] [answered]
The last configuration file  manifest single deploy for Kong OSS on K8S with the Kong Manager OSS does not exist at https://github.com/Kong/kubernetes-ingress-controller/blob/main/deploy/single /  https://docs.konghq.com/gateway/3.4.x/install/kubernetes/kubectl/  . Similarly, in the Helm options, there is no option available for Kong OSS with the Kong Manager OSS,  https://docs.konghq.com/gateway/3.4.x/install/kubernetes/helm-quickstart/.

Could you please provide the configuration file  manifest  single deploy for Kong OSS on K8S with the Kong Manager OSS ?

**Comment by sumimakito** [comment, marked as answer]
Hello @bilel-zouaghi, thanks for bringing this issue up. 

We have not provided an official example of setting up Kong Gateway OSS with Kong Manager OSS using Helm charts in our documentation. However, Kong's Helm charts have [a `manager` option](https://github.com/Kong/charts/blob/dfaba282a9dbe259a3bd4214caaca637d7ff4426/charts/kong/values.yaml#L1029-L1074) to enable the Kubernetes service for Kong Manager.

While we are working on updating our documentation, we recommended exploring Kong Manager with Helm charts instead of KIC's all-in-one manifests.
